x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/index_pattern_prompt/index_pattern_prompt.tsx

```diff
@@ -34,7 +34,7 @@ export const IndexPatternPrompt: FC<Props> = ({ destIndex, color }) => {
       <EuiText size="xs" color={color ?? 'warning'}>
         <FormattedMessage
           id="xpack.ml.dataframe.analytics.dataViewPromptMessage"
-          defaultMessage="No data view exists for index{destIndex}. "
+          defaultMessage="No data view exists for index {destIndex}. "
```

Comment:
This will leave an extra space before the period if the destIndex is undefined. "No data view exists for index ."